python-template/{{cookiecutter.placeholder_repo_name}}/.gitignore

```diff
@@ -0,0 +1,78 @@
+*.py[cod]
+__pycache__
+.pytest_cache
+
+# C extensions
+*.so
+
+# Packages
+*.egg
+*.egg-info
+dist
+build
+eggs
+parts
+bin
+var
+sdist
+develop-eggs
+.installed.cfg
+lib
+lib64
+
+# Installer logs
+pip-log.txt
+
+# Unit test / coverage reports
+.cache/
+.pytest_cache/
+.coverage
+.coverage.*
+.tox
+coverage.xml
+htmlcov/
+
+
+
+# Translations
+*.mo
+
+# IDEs and text editors
```

Comment:
My preference is for people to use their own `.gitignore-global` files for these, since different people use different editors and operating systems. (Applies to IDE and OS sections.)

Reply:
I agree with that